src/util/innerStore.ts

```diff
@@ -70,7 +70,19 @@ export const setCommonECData = (seriesIndex: number, dataType: SeriesDataType, d
                 childECData.seriesIndex = seriesIndex;
                 childECData.dataIndex = dataIdx;
                 childECData.dataType = dataType;
+                child.__metaData = {
```

Comment:
`setCommonECData` is probably not designed for only "seriesItem", see "progress" in GaugeView.
And I am worried that it's not a good idea to give each el a extra object to save seriesIndex and dataIndex, particularly the already save in ecData.

Could we move `makeInner` to zrender, and create `getECData` in zrender (with another name) and visit the "ecData" in zrender to  get seriesIndex and dataIndex?